src/plugins/intel_gpu/tests/test_cases/reduce_gpu_test.cpp

```diff
@@ -1641,6 +1641,74 @@ TEST(reduce_gpu, common_bfwzyx_log_sum_exp_keepdims) {
     }
 }
 
+TEST(reduce_gpu, b_fs_yx_fsv16_min_dynamic) {
+    auto& engine = get_test_engine();
+    auto input = engine.allocate_memory({data_types::f32, format::b_fs_yx_fsv16, {1, 1, 1, 1}});
+
+    set_values(input, {1.0f});
```

Comment:
Could you make this test more meaningful? only one data is not good to test minimum..
Also to check blocked format we'd better set the feature size > 16

Reply:
Updated